# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/DMABufSurface.cpp
# Commit: bf6209fd39e8
# Full Hash: bf6209fd39e85664974d19bfc1c8cb6d06d48a21
# Author: stransky <stransky@redhat.com>
# Date: 2025-02-05 16:56:59
# Regressor Bug: 1944490
# File Overlap Count: 1
# Description:
#   Bug 1944490 [Linux] Ensure DMABufFormats provides basic dmabuf formats r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D236688
# ==============================================================================

diff -r a374b7104d05 -r bf6209fd39e8 widget/gtk/DMABufSurface.cpp
--- a/widget/gtk/DMABufSurface.cpp	Wed Feb 05 11:23:30 2025 +0000
+++ b/widget/gtk/DMABufSurface.cpp	Wed Feb 05 11:26:53 2025 +0000
@@ -557,6 +557,8 @@
                                                      : GBM_FORMAT_XRGB8888;
   RefPtr<DRMFormat> format = GetDMABufDevice()->GetDRMFormat(mFOURCCFormat);
   if (!format) {
+    LOGDMABUF(("DMABufSurfaceRGBA::Create(): Missing drm format 0x%x!",
+               mFOURCCFormat));
     return false;
   }
   return Create(aWidth, aHeight, format, aDMABufSurfaceFlags);